UOP1_Project/.editorconfig

```diff
@@ -1,4 +1,5 @@
 [*.cs]
+end_of_line = crlf
```

Comment:
Windows/Linux and VisualStudio/Rider have different line endings by default. So I installed `crlf` here.

Reply:
This doesn't belong to this PR though. Can you remove it and put it in a different one?
Also not super sure how the repo will react. Will all existing files be modified? I'm not an expert on this.

Reply:
Done